App/Config/ReduxPersist.ts

```diff
@@ -0,0 +1,82 @@
+import { AsyncStorage } from 'react-native'
+import { createMigrate, PersistConfig, MigrationManifest } from 'redux-persist'
+
+const migrations: MigrationManifest = {
```

Comment:
Still need to figure out the right way to be able to use TS types during data migration. For now, just casting to `any` let's us pretend it's plain JS objects.